web/src/components/Settings.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <ToolBar>
+      <v-tooltip bottom>
+        <template #activator="{ on, attrs }">
+          <v-btn v-bind="attrs" icon v-on="on" @click="getSettings">
+            <v-icon>mdi-refresh</v-icon>
+          </v-btn>
+        </template>
+        <span>Refresh</span>
+      </v-tooltip>
+    </ToolBar>
+    <v-card>
+      <v-card-title>Settings</v-card-title>
+      <v-simple-table>
+        <tbody>
+          <tr v-for="(value, name) in store.clientConfig || []" :key="name">
+            <th>{{ name }}</th>
+            <td width="100%">
+              <input v-model="visible" type="checkbox" @change="save" />
+            </td>
+          </tr>
+        </tbody>
+      </v-simple-table>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import ToolBar from "./ToolBar.vue";
+import { onMounted } from "vue";
+import { useRootStore } from "@/stores";
+import { EventBus } from "./EventBus";
+import { ref, computed } from "vue";
+
+const store = useRootStore();
+const visible = ref(false);
```

Comment:
`visible` is an odd name for the value of that config value. This will break too when the clientConfig has more than one field because all would share the same model, but that could be handled once there actually are multiple config options :D